src/program.ts

```diff
@@ -24,11 +24,11 @@ export type Program<Init, State, Action, View> = {
  * @typeParam Init This is the type of the initial value passed to the program's init function.
  * @category Program
  */
-export function run<Init, State, ProgramAction, View>(
+export function run<Init, State, ProgramAction, View, ManagerAction>(
```

Comment:
There are multiple effect managers, each with their own separate Action type. So I don't think it is possible to type to a single effect manager action for the program.

Reply:
Case in point, the `run` function is part of the external facing API.